best-practices/README.md

```diff
@@ -189,10 +189,9 @@ Email
 
 JavaScript
 ----------
-* Use Coffeescript, ES6 with [babel], or another language that compiles to
-  JavaScript
+* Use the latest stable JavaScript syntax with a transpiler, such as [babel].
```

Comment:
Saying "with a transpiler" seems unnecessary; either the latest JS syntax is supported by all our browsers, in which case we don't need a compiler, or it isn't, in which case we do. This seems like a judgement that the team will have to make for the project based on the target JS runtimes.